app/javascript/dashboard/routes/dashboard/settings/sla/components/Toggle.vue

```diff
@@ -0,0 +1,36 @@
+<script setup>
+import { ref } from 'vue';
+
+const props = defineProps({
+  value: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const emit = defineEmits(['input']);
+const isChecked = ref(props.value);
+
+const toggleCheckbox = () => {
+  isChecked.value = !isChecked.value;
+  emit('input', isChecked.value);
+};
+</script>
+
+<template>
+  <label class="relative inline-block h-4 w-7">
+    <input
+      class="hidden peer"
+      type="checkbox"
+      :checked="isChecked"
+      @click="toggleCheckbox"
+    />
+    <div
+      :class="[
```

Comment:
@scmmishra Can you add dark mode support for this toggle switch.
<img width="167" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/af760185-36de-44e1-9844-53187b7e18b7">